frontend/public/components/console-notifier.tsx

```diff
@@ -16,7 +16,10 @@ const ConsoleNotifier_: React.FC<ConsoleNotifierProps> = ({ obj, location }) =>
     <>
       {_.map(_.get(obj, 'data'), (notification) =>
         notification.spec.location === location ||
-        notification.spec.location === 'BannerTopBottom' ? (
+        notification.spec.location === 'BannerTopBottom' ||
+        // notification.spec.location is optional, so always output the notification
+        // if location is not specified, producing the same result as location: BannerTopBottom
```

Comment:
I'm OK with this, but I have a slight preference for defaulting to `BannerTop`. I think that's usually what people want.

Reply:
I went with this because it was the simplest code change...

Reply:
Updated to default location to `BannerTop` if `location` not specified.